import `a` conflicts with prior import of same name